java/lib/CameraServerJNI.cpp

```diff
@@ -225,9 +217,8 @@ extern "C" {
  * Method:    getPropertyKind
  * Signature: (I)I
  */
-JNIEXPORT jint JNICALL Java_edu_wpi_cscore_CameraServerJNI_getPropertyKind
-  (JNIEnv *env, jclass, jint property)
-{
+JNIEXPORT jint JNICALL Java_edu_wpi_cscore_CameraServerJNI_getPropertyKind(
+    JNIEnv* env, jclass, jint property) {
```

Comment:
I'll fix these then add this to the generated files list. A clang-format touch-up task that does the JNI formatting properly is on my todo list.